test/performance/odp_sched_pktio.c

```diff
@@ -1049,17 +1062,30 @@ static void link_pktios(test_global_t *test_global)
 
 static int start_pktios(test_global_t *test_global)
 {
-	int i;
+	test_options_t *test_options = &test_global->opt;
+	uint32_t num_pktio = test_options->num_pktio;
+	uint32_t wait_sec = test_options->wait_sec;
+	odp_pktio_t pktios[num_pktio];
+	const char *names[num_pktio];
+	uint32_t i;
 
-	for (i = 0; i < test_global->opt.num_pktio; i++) {
+	for (i = 0; i < num_pktio; i++) {
 		if (odp_pktio_start(test_global->pktio[i].pktio)) {
-			printf("Error (%s): Pktio start failed.\n",
-			       test_global->opt.pktio_name[i]);
-
+			ODPH_ERR("Error (%s): Pktio start failed.\n", test_options->pktio_name[i]);
 			return -1;
 		}
 
 		test_global->pktio[i].started = 1;
+		pktios[i] = test_global->pktio[i].pktio;
+		names[i] = test_options->pktio_name[i];
+	}
+
+	/* Wait until all links are up and print link infos */
+	if (wait_sec) {
+		if (pktio_common_check_link_status_wait(pktios, names, num_pktio, wait_sec) == -1)
+			return -1;
+
+		pktio_common_print_link_info_multi(pktios, names, num_pktio);
```

Comment:
Perhaps this info printing could be done even when wait_sec is zero (and same in other commits)? The commit message could mention that link info printing is also being added.